packages/moonstone/TooltipDecorator/TooltipDecorator.js

```diff
@@ -346,13 +277,16 @@ const TooltipDecorator = hoc(defaultConfig, (config, Wrapped) => {
 
 		handle = handle.bind(this)
 
-		handleKeyDown = this.handle(
-			forProp('disabled', true),
-			() => {
+		handleKeyDown = (ev) => {
+			forward('onKeyDown', ev, this.props);
```

Comment:
We shouldn't forward local event callbacks from a global handler since the events might not have originated from this DOM tree.